src/aggregator/aggregator/aggregator_test.go

```diff
@@ -984,34 +984,44 @@ func TestAggregatorOwnedShards(t *testing.T) {
 	}
 }
 
-func TestAggregatorAddMetricMetrics(t *testing.T) {
+func TestAggregatorAddUntimedMetrics(t *testing.T) {
```

Comment:
I know this is more a comment towards the original code, but would it be worth adding similar test cases for passthrough and forwarded metrics as well?

Reply:
https://github.com/m3db/m3/pull/3559/commits/f7b68c2758f72dcf2b13d1eb4ed6b71d2c11fd63